F acti management seminar

duonglnn opened the pull request:
- Change color & font for header
- Create Activities Management : Seminar + Club

duonglnn committed:
- Change color + font header dropdown
- Create seminar + club activities management

atulaa1 commented on src/app/@theme/components/header/header.component.html:
```diff
@@ -62,7 +62,7 @@
               class="text-center font-weight-600">
             <i style="font-size: 23px;" class="fas fa-users"></i>
-            <a style="margin-left: 10px" ng-reflect-router-link="" title="Logout" href="#/pages/dashboard">
+            <a style="margin-left: 10px" ng-reflect-router-link="" title="Logout" href="#/pages/acti-management">
```
sửa title đúng với menu

atulaa1 commented on src/app/@theme/components/header/header.component.scss:
```diff
@@ -233,3 +233,7 @@
 }
+
+a span {
+  color: #01dbb5;
```
lấy mã mầu đồng nhất với màu của main button và màu border

atulaa1 commented on src/app/app.module.ts:
```diff
@@ -30,10 +32,10 @@ import { LoginComponent } from './modals/login/login.component';
   ],
   bootstrap: [AppComponent],
-  providers: [BsModalService, BsModalRef,
+  providers: [BsModalService, BsModalRef, NgbActiveModal,
```
thừa NgbActiveModal

duonglnn committed:
- Change color + font header dropdown
- Change title for Activities Management
- Create seminar + club + team building activities management

atulaa1 reviewed (approved):
cần sửa nốt các title của các menu còn lai
